examples/foundry/image_classification.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<p>I think we need to explain this a bit better:</p><p>Something like: Mapping a model's ontology to a project's ontology is currently not supported through the SDK, however, to showcase how to send foundry predictions to a project, we are going to generate the mapping of the foundry app ontology to the project ontology through the UI.</p><p> </p><p> </p><ol><li>Navigate to your dataset you created for your model run</li><li>Select&nbsp;<strong>Select all</strong>&nbsp;in the top right corner</li><li>Select&nbsp;<strong>Manage selection</strong>&nbsp;&gt;&nbsp;<strong>Send to Annotate</strong></li><li>Specify the project we created from the project dropdown menu</li><li>Selecting a workflow step is not required since we are not sending annotations to a project using this UI modal.</li><li>Mark&nbsp;<strong>Include model predictions</strong>&nbsp;then scroll down and select&nbsp;<strong>Map</strong></li><li>Select the incoming ontology and matching ontology feature for both Computer and Phone</li><li>Once both features are mapped press the&nbsp;<strong>Copy ontology mapping as JSON</strong>&nbsp;in the top right corner</li><li>Do not save this configuration, since we are not sending predictions to a project using this UI modal. We will be sending predictions in the following steps using the SDK. </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>